docs/_data/prague-2020-config.yaml

```diff
@@ -0,0 +1,138 @@
+name: Prague
+
+shortcode: prague # prague australia?
+year: 2020
+city: Prague
+local_area: Czech Republic
+area: Europe
+area_adj: European
+tz: CEST
+email: prague@writethedocs.org
+color: coral
+
+photos:
+  default: _static/conf/images/headers/prague-group.png
+
+buttons:
+  top:
+     - text: Sponsor the conference
+       link: /sponsors/prospectus
+    # - text: Submit a Talk
+    #  link: /cfp
+    # - text: Buy a Ticket!
+    #   link: /tickets
+    # - text: See the talks!
+    #   link: /speakers
+#    - text: See the Schedule!
+#      link: /schedule
+    # - text: Welcome Wagon
+    #   link: /welcome-wagon
+#    - text: Watch the Live Stream
+#      link: /livestream
+#     - text: Browse the photos
+#       link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+#     - text: Watch the videos!
+#       link_absolute: /videos/portland/2020
+  bottom:
+     - text: Sponsor the conference
+       link: /sponsors/prospectus
+#     - text: Submit a Talk
+#       link: /cfp
+    # - text: Buy a Ticket!
+    #   link: /tickets
+#    - text: Prepare with the Welcome Wagon
+#      link: /welcome-wagon
+#    - text: See the schedule!
+#      link: /schedule
+    # - text: Watch the Live Stream
+    #  link: /livestream
+#     - text: Browse the photos
+#       link_absolute: https://www.flickr.com/photos/writethedocs/albums/72157691507514803
+#     - text: Watch the videos!
+#       link_absolute: /videos/portland/2020
+
+tickets:
+  community:
+    price: €600
+  corporate:
+    price: €600
+  independent:
+    price: €175
+  student:
+    price: €75
+
+sponsorship:
```

Comment:
@ericholscher how are we handling sponsorship prices? The prospectus should be up already with the landing page...

Reply:
I'm inclined to leave them similar, since we don't have as much sponsorship in Prague as we do in Portland. If we're forecasting a higher attendee count tho, we should raise them some. Maybe bump 20% to account for more humans?

Reply:
@ericholscher it's up to you and I trust whatever you decide, but if we want to publish this before the holidays we should decide today if possible...

Reply:
ok I adjusted based on Portland prices rounding down a bit